package.json

```diff
@@ -95,6 +95,7 @@
     "react-dom": "^17.0.2",
     "single-spa": "^5.9.3",
     "single-spa-react": "^4.4.0",
+    "query-string": "^7.1.1",
```

Comment:
This looks like it may no longer be needed

Reply:
Agreed. Removed.